src/Stratis.Bitcoin.Features.Api/Startup.cs

```diff
@@ -97,6 +98,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseCors("CorsPolicy");
 
+            app.UseStaticFiles(new StaticFileOptions
+            {
+                FileProvider = new PhysicalFileProvider(@"C:\Users\me\Stratis\StratisBitcoinFullNode\src\Stratis.Bitcoin.Features.Api\wwwroot\"),
```

Comment:
I need some help to found the `Stratis.Bitcoin.Features.Api` folder programatically, but for now I add it as raw string

Reply:
Done (in bc0580ae6b2b4f889d48c3bb0a0acd57467da78b )